<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWt</class>
 <widget class="QMainWindow" name="MainWt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>463</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe Print</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Plastic Types</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/icoo.png</normaloff>../../../Downloads/icoo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {

    background-color: lightblue;
	color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.136364 rgba(63, 85, 255, 27), stop:0.288136 rgba(118, 255, 95, 21), stop:0.519774 rgba(95, 127, 255, 21), stop:0.858757 rgba(95, 248, 255, 20));
    border: 2px groove gray;
     border-radius: 20px;
    padding: 2px 4px;
    border: 2px groove DarkCyan;
}
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>Polyethylene terephthalate</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: white;
   border: 2px groove DarkCyan;
    border-radius: 10px;
   padding: 2px 4px;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>PET</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Desktop/1.jpg</normaloff>../../Desktop/1.jpg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>High-density polyethylene</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: white;
   border: 2px groove DarkCyan;
    border-radius: 10px;
   padding: 2px 4px;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>HDPE</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Desktop/2.jpg</normaloff>../../Desktop/2.jpg</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>99</y>
      <width>121</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     border-radius: 10px;
    padding: 2px 4px;
   	background-color: white;
   border: 2px groove DarkCyan;
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>Verdana</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>or V (Polyvinyl chloride)</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     border-radius: 10px;
    padding: 2px 4px;
   	background-color: white;
   border: 2px groove DarkCyan;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>PVC</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Desktop/3.jpg</normaloff>../../Desktop/3.jpg</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Verdana</family>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Low density polyethylene</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     border-radius: 10px;
    padding: 2px 4px;
   	background-color: white;
   border: 2px groove DarkCyan;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>LDPE</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Desktop/4.jpg</normaloff>../../Desktop/4.jpg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>Verdana</family>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Polypropylene</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     border-radius: 10px;
    padding: 2px 4px;
   	background-color: white;
   border: 2px groove DarkCyan;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>PP</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Desktop/5.jpg</normaloff>../../Desktop/5.jpg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Verdana</family>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="statusTip">
        <string>Polystyrene</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     border-radius: 10px;
    padding: 2px 4px;
   	background-color: white;
   border: 2px groove DarkCyan;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}
</string>
       </property>
       <property name="text">
        <string>PS</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Desktop/6.jpg</normaloff>../../Desktop/6.jpg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>Verdana</family>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
     border-radius: 10px;
    padding: 2px 4px;
   	background-color: white;
   border: 2px groove DarkCyan;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>Others</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Desktop/7.jpg</normaloff>../../Desktop/7.jpg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>411</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLable{

	color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.318182 rgba(118, 255, 95, 156), stop:0.534091 rgba(63, 85, 255, 156), stop:0.732955 rgba(95, 248, 255, 183), stop:0.886364 rgba(95, 127, 255, 156));
    border: 2px groove DarkCyan;
     border-radius: 20px;
    padding: 2px 4px
   }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/loop_adobespark.png</pixmap>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>371</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser {
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.125 rgba(63, 85, 255, 156), stop:0.318182 rgba(118, 255, 95, 156), stop:0.732955 rgba(95, 248, 255, 183), stop:0.886364 rgba(95, 127, 255, 156));

    border: 2px groove DarkCyan;
     border-radius: 20px;
    padding: 2px 4px
   }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: white;
    border: 2px groove DarkCyan;
     border-radius: 20px;
    padding: 2px 4px
   }

font: 8pt &quot;Verdana&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.318182 rgba(118, 255, 95, 156), stop:0.534091 rgba(63, 85, 255, 156), stop:0.732955 rgba(95, 248, 255, 183), stop:0.886364 rgba(95, 127, 255, 156));
    border-radius: 10px;
    padding: 2px 4px;
    border: 2px groove DarkCyan;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
